cumulus/parachains/runtimes/assets/asset-hub-rococo/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,172 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Asset Hub Rococo Runtime genesis config presets
+
+use alloc::{vec, vec::Vec};
+use cumulus_primitives_core::ParaId;
+use hex_literal::hex;
+use parachains_common::{genesis_config_helpers::*, AccountId, AuraId, Balance as AssetHubBalance};
+use sp_core::{crypto::UncheckedInto, sr25519};
+use testnet_parachains_constants::rococo::xcm_version::SAFE_XCM_VERSION;
+
+const ASSET_HUB_ROCOCO_ED: AssetHubBalance = crate::ExistentialDeposit::get();
+
+/// Generate the session keys from individual elements.
+///
+/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
```

Comment:
Not sure about this comment and naming, why exactly will this change to a tuple?

Reply:
Just moved that function from other place w/o thinking too much about it:
https://github.com/paritytech/polkadot-sdk/blob/9cdf3d999e9de7996495c891af7085f55acc695a/cumulus/polkadot-parachain/src/chain_spec/asset_hubs.rs#L37